lib/index.js

```diff
@@ -2,7 +2,7 @@
 // https://github.com/markdown-it/markdown-it
 const path = require('path')
 
-module.exports = function nuxtMarkdownit (options) {
+module.exports = function nuxtMarkdownit(options) {
```

Comment:
It would be also nice if we migrated module to typescript (new module template) with same PR but not necessary can do later :)

Reply:
I will try to do my best to migrate it entirely in Typescript 😀